spec/setup_acceptance_node.pp

```diff
@@ -0,0 +1,48 @@
+# The omnibus installer use the following algorithm to know what to do.
```

Comment:
See https://github.com/voxpupuli/voxpupuli-acceptance/#per-host-configuration for how this file is used.